packages/react/src/components/Popover/Popover-story.js

```diff
@@ -0,0 +1,442 @@
+        <section>
+          <h2>Popover Visibility</h2>
+          <div>
+            <label>
+              <input
+                type="checkbox"
+                value={open}
+                checked={open}
+                onChange={() => {
+                  setOpen(!open);
+                }}
+              />
+              Open
+            </label>
+          </div>
+        </section>
+        <div style={{ marginTop: '2rem' }}>
+          <Popover open={open} align={align} relative>
+            <PopoverContent className="p-3">Hello</PopoverContent>
+          </Popover>
+        </div>
+      </>
+    );
+  }
+
+  return <PopoverDemo />;
+};
+
+export const Playground = () => {
+  const options = [
+    'top',
+    'top-left',
+    'top-right',
+
+    'bottom',
+    'bottom-left',
+    'bottom-right',
+
+    'left',
+    'left-bottom',
+    'left-top',
+
+    'right',
+    'right-bottom',
+    'right-top',
+  ];
+
+  return (
+    <Popover
+      align={select('Specify the caret alignment', options, 'top')}
+      caret={boolean('Specify whether the caret should be visible', true)}
+      highContrast={boolean(
+        'Specify whether the high-contrast variant should render',
+        false
+      )}
+      light={boolean('Specify whether the light variant should render', false)}
+      open={boolean('Specify whether the popover is open or closed', true)}>
+      <PopoverContent className="p-3">Sample content</PopoverContent>
+    </Popover>
+  );
+};
+
+Playground.story = {
+  decorators: [
+    (story) => (
+      <div className="mt-9">
+        <div className="position-relative">{story()}</div>
+      </div>
+    ),
+  ],
+};
+
+export const Examples = () => {
+  function Example({
+    children,
+    align,
+    relative = false,
+    icon,
+    size,
+    flip = false,
+    ...rest
+  }) {
+    const ref = React.useRef(null);
+    const [open, setOpen] = React.useState(false);
+
+    useOutsideClick(ref, () => {
+      if (open === true) {
+        setOpen(false);
+      }
+    });
+
+    if (flip) {
+      return (
+        <div ref={ref} {...rest}>
+          <Popover align={align} open={open} relative={relative}>
+            <PopoverContent className="p-3">{children}</PopoverContent>
+          </Popover>
+          <Button
+            iconDescription="Toggle"
+            hasIconOnly
+            onClick={() => {
+              setOpen(!open);
+            }}
+            renderIcon={icon}
+            size={size}
+          />
+        </div>
+      );
+    }
+
+    return (
+      <div ref={ref} {...rest}>
+        <Button
+          iconDescription="Toggle"
+          hasIconOnly
+          onClick={() => {
+            setOpen(!open);
+          }}
+          renderIcon={icon}
+          size={size}
+        />
+        <Popover align={align} open={open} relative={relative}>
+          <PopoverContent className="p-3">{children}</PopoverContent>
+        </Popover>
+      </div>
+    );
+  }
+
+  return (
```

Comment:
Can all this extra stuff in the story file be removed?

Reply:
I figured we could keep it for now until we figure out a better alignment story that we can use to verify (or only enable ti for development but not for production kind of thing)